app/models/question.rb

```diff
@@ -1,4 +1,5 @@
 class Question < ActiveRecord::Base
 	belongs_to :topic
```

Comment:
use 2 spaces indentation
+ good approach to not use validates for foreign fields *_id
try to find another way how you can write this code